curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-form-element.md

```diff
@@ -37,9 +37,7 @@ assert(
 Your `form` should have an `action` attribute which is set to `https://www.freecatphotoapp.com/submit-cat-photo`
 
 ```js
-assert(
-  $('form').attr('action') === 'https://www.freecatphotoapp.com/submit-cat-photo'
-);
+assert(code.match(/https:\/\/(www\.)?freecatphotoapp\.com\/submit-cat-photo/i))
```

Comment:
This change is a regression in the test, IMHO - it'll mean we aren't testing the form action attribute at all, only that the url is somewhere in the code.

What about:
```suggestion
const action = $('form').attr('action');
assert(action.match(/https:\/\/(www\.)?freecatphotoapp\.com\/submit-cat-photo/i))
```